### DISCUSSION TITLE
Adding a delete all button within activites

### DISCUSSION THREAD

**Discussion by jason-feng** [ghostfolio/ghostfolio#1727] [Ideas]
<img width="1483" alt="image" src="https://user-images.githubusercontent.com/7146007/219903033-1f99a9c4-fc0a-45db-9da2-50e672aefe50.png">

I thought this would be a nice feature, especially when people are still testing sample data. What do y'all think?

**Comment by KarelBeyr** [comment]
I solved similar issue in self hosting environment by deleting what I didn't want directly in the database.
